docker-compose.test.yml

```diff
@@ -4,6 +4,8 @@ services:
   backend:
     build:
       dockerfile: Dockerfile.test
+    depends_on:
+      - neo4j_db
```

Comment:
Ist eigentlich schon in der docker-compose.yml definiert, deswegen muss es hier nicht nochmal rein